### PR TITLE
increase width of 3 column headshot view

### PR DESCRIPTION
**Pull request by tanyaparker**
| Before | After |
| :-: | :-: |
| screen shot 2016-09-14 at 1 19 09 pm | screen shot 2016-09-14 at 1 19 16 pm |
| screen shot 2016-09-14 at 1 19 22 pm | screen shot 2016-09-14 at 1 19 29 pm |
| screen shot 2016-09-14 at 1 18 40 pm | screen shot 2016-09-14 at 1 18 36 pm |
| screen shot 2016-09-14 at 1 18 54 pm | screen shot 2016-09-14 at 1 18 58 pm |

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/about_headshots.haml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-#headshots{:style=>'width: 760px; padding: 0; margin: 0;'}
+#headshots{:style=>'width: 790px; padding: 0; margin: 0;'}
   - people.each_slice(columns) do |group|
-    .headshot_row{:style=>'width: 760px; padding: 0; margin: 0;'}
+    .headshot_row{:style=>'width: 790px; padding: 0; margin: 0;'}
       - group.each do |person|
-        - column_width = columns==3 ? '220' : '160'
+        - column_width = columns==3 ? '260' : '160'
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/views/about_headshots.haml:5]: We could do 263px but I like this round number.

**Comment by breville** [comment]
should we make the headshots even bigger?

**Comment by tanyaparker** [comment]
hehe you're right, they do look tiny.

100px
image

150px
image

175px
image

200px
image

**Comment by breville** [comment]
I kinda like the biggest size.  Two issues are whether source images are always high enough resolution, and whether it makes the page too long, but I think these days it's all about big photos.

**Comment by tanyaparker** [comment]
It looks pretty good on our team page but there are a few low res images on some of the other pages.

**Comment by tanyaparker** [comment]
I'm going to go with 200. Also just talked with Caley. We like how it feels more put together; the bigger image makes each person's pic + bio feel like one block. And for the low-res images they're still not _that bad_ at 200.

**Comment by tanyaparker** [comment]
What do you think about circles?

image

**Comment by breville** [comment]
Ah, speaking of major trends. :)  They probably work well, but we should think about how we use them across the site, and presumably use them for headshots of people.  Do we have other obvious pages that would get them too?  Do they work with our brand?

**Comment by tanyaparker** [comment]
All the pages that use this view are in the table above, so you can see their context. I think this feels more modern and we use this style with our 3 people quotes like [code.org/about/2014](code.org/about/2014).

**Comment by breville** [comment]
In that case, worth a shot.  Presumably it's all done in CSS so source files stay the same.

**Pull request merged**